dashboard/src2/pages/BillingInvoices.vue

```diff
@@ -109,13 +109,13 @@ export default {
 							}
 							return 'Prepaid Credits';
 						},
-						width: 0.5
+						width: 0.8
 					},
 					{
 						label: 'Status',
 						fieldname: 'status',
 						type: 'Badge',
-						width: '150px'
+						width: 0.5
```

Comment:
These are better off with a fixed width since badge has a capped width (the longest badge)